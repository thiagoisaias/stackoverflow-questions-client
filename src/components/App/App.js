import React, { Component } from "react";
import Search from "../Search/Search";

import { LayoutWrapper } from "../../shared/styled";

class App extends Component {
  render() {
    return (
      <LayoutWrapper>
        <Search />
      </LayoutWrapper>
    );
  }
}

export default App;
