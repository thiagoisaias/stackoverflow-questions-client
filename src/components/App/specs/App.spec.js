import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import App from "../App";

describe("App", () => {
  const component = shallow(<App />);

  it("renders properly", () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
