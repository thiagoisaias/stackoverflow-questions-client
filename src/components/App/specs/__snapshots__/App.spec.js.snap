// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<styled.div>
  <Search />
</styled.div>
`;
